### DISCUSSION TITLE
2 factor authentication

### DISCUSSION THREAD

**Discussion by AntonBuyan** [deepgram/community#848] [Feature Requests]
Not sure I see it, how to add 2nd authentication method to secure the account beyond just username/pw?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @AntonBuyan, at this time we do not offer 2-factor authentication. We only offer username/password, or SSO with Google, GitHub, or Azure.

**Comment by YanivAspis** [comment]
Bringing this discussion back up because we also require multi-factor authentication for compliance reasons.
The auditors flagged this as an issue so without this we will have to remove DeepGram integration from our app.
Any chance this can be added? With an option to enforce MFA on all members of the team.

**Reply by jkroll-deepgram** [reply]
Hi @YanivAspis, we still do not offer multi-factor authentication, and don't currently have plans to add it. Customers may choose to restrict Deepgram Console access to a limited set of administrative users, since technical users typically can perform all needed actions through our APIs (such as making API requests, and using our management API). Perhaps that is an option you can consider. If not, is there a particular type of certification you need to pass, or is it due to your company's security policies?

**Reply by YanivAspis** [reply]
Hi @jkroll-deepgram, thanks for replying. This is for ISO 27001 certification, and we've been specifically asked to ensure MFA is enabled for all users, especially those with privileged access such as administrators. Is there any chance this feature can be considered?

**Reply by jkroll-deepgram** [reply]
Thank you for the additional context, @YanivAspis. I checked in with our information security team. Their recommendation is to use our SSO for signing up for Deepgram Console accounts. We support signing in with Google, GitHub, and Azure. If you then require MFA to log into one of those platforms within your company, then the MFA requirement is met. Would that suffice?

**Reply by YanivAspis** [reply]
Thanks @jkroll-deepgram for checking with the team, we'll set up SSO in lieu of MFA and check with our auditors if this will suffice.
